cheatsheets/Cryptographic_Storage_Cheat_Sheet.md

```diff
@@ -63,12 +63,13 @@ Ensure that random algorithms are seeded with sufficient entropy.
 
 Tools like [NIST RNG Test tool](http://csrc.nist.gov/groups/ST/toolkit/rng/documentation_software.html) (as used in PCI PTS Derived Test Requirements) can be used to comprehensively assess the quality of a Random Number Generator by reading e.g. 128MB of data from the RNG source and then assessing its randomness properties with the tool.
 
-The following libraries are considered **weak** random numbers generators and should not be used.
+The following functions are considered **weak** random numbers generators and should not be used.
 
-- C library: `random()`, `rand()` instead use [getrandom(2)](http://man7.org/linux/man-pages/man2/getrandom.2.html) instead
-- Java library: `java.util.Random()` instead use `java.security.SecureRandom` instead
+- C : `random()`, `rand()` instead use [getrandom(2)](http://man7.org/linux/man-pages/man2/getrandom.2.html)
+- Java : `java.util.Random()` instead use `java.security.SecureRandom`
+- PHP : `rand()`, `mt_rand()`, `array_rand()`, `uniqid()` instead use `random_bytes()`, `random_int()` (PHP 7) or `openssl_random_pseudo_bytes()` (PHP 5)
```

Comment:
Remove PHP 5 as it is deprecated.
Is it possible to add links to the functions for direct reference?

Reply:
It can be interesting to keep PHP5 (and mention that it is deprecated in bold) at least for people that are stuck with the migration to PHP7 or for legacy PHP application that are still maintained but no migrated.

Reply:
Added links and a note that PHP 5 is deprecated and should not be used.